lib/multi-level.js

```diff
@@ -17,11 +38,11 @@
 /**
  * A multi-level cache.
  *
- * @param {Object} opts                             - Options for the Client instance
- * @param {String} [opts.maxAge=600,000]            - The duration, in milliseconds, before a cached item expires
- * @param {String} [opts.maxStaleness=0]            - The duration, in milliseconds, in which expired cache items are still served
- * @param {ShouldCache} [opts.shouldCache=()=>true] - Function to determine whether to not we should cache the result
- * @param {Cache[]}  opts.caches                    - An Array of cache objects. See `./fs` and `./memory` for sample caches.
+ * @param {Object} opts Options for the Client instance
+ * @param {Number} [opts.maxAge=600,000] The duration, in milliseconds, before a cached item expires
```

Comment:
The comma should probably be removed here, as it isn't in the actual default. (We might want to include a version of the default with the comma in the description for readability).